src/File.php

```diff
@@ -55,6 +55,9 @@
  */
 class File
 {
+
+    const DEFAULT_HTTP_ACCEPT = 'application/atom+xml, application/rss+xml, application/rdf+xml;q=0.9, application/xml;q=0.8, text/xml;q=0.8, text/html;q=0.7, unknown/unknown;q=0.1, application/unknown;q=0.1, */*;q=0.1';
```

Comment:
I suggest that we define this constant as `DEFAULT_HTTP_ACCEPT_HEADER` in the `SimplePie` class and mark it as `@internal`, since the `File` class will soon be deprecated.

Reply:
Done